dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/handlers/AndroidSpanLogsHandler.kt

```diff
@@ -85,7 +85,7 @@ internal class AndroidSpanLogsHandler(
 
         if (errorType != null) {
             val stackField = map.remove(Fields.STACK)
-            val msgField = map.remove(Fields.MESSAGE)
+            val msgField = map[Fields.MESSAGE]
```

Comment:
I think the reason it was like that is that I followed what they are doing in the APM project.